src/multivariate.jl

```diff
@@ -90,17 +90,22 @@ end
 
 A multivariate Normal distribution whose covariance is dense. Compatible with Tracker.
 """
-struct TuringDenseMvNormal{Tm<:AbstractVector, TC<:Cholesky} <: ContinuousMultivariateDistribution
+struct TuringDenseMvNormal{Tm<:AbstractVector, TC<:AbstractMatrix} <: ContinuousMultivariateDistribution
     m::Tm
     C::TC
 end
 function TuringDenseMvNormal(m::AbstractVector, A::AbstractMatrix)
-    return TuringDenseMvNormal(m, cholesky(A))
+    return TuringDenseMvNormal(m, cholesky(A).L)
 end
+
+function TuringSqrtDenseMvNormal(m::AbstractVector, C::AbstractMatrix)
+    TuringDenseMvNormal(m, C)
```

Comment:
I don't see how this could work? It will use the constructor above and perform a Cholesky decomposition anyways?

Reply:
Oh you're right, the constructor is wrong, the following would work directly no ? 
```suggestion
    TuringDenseMvNormal{typeof(m), typeof(C)}(m, C)
```